benchmarks/benchmarks.py

```diff
@@ -4,10 +4,21 @@
 
 from memray import AllocatorType
 from memray import FileReader
-from memray import MemoryAllocator
+
+try:
+    from memray._test import MemoryAllocator
+except ImportError:
+    from memray import MemoryAllocator
```

Comment:
Hm? Is this for compatibility across versions of Memray? Any reason to still support the old one?

Reply:
Yep, `asv` us used to benchmark across all it history so it must be able to run reasonably well in all commits. (is ok if **some** tests don't run, `asv` handles that) .